app/javascript/containers/AdminPreviewStatic.js

```diff
@@ -0,0 +1,151 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import Cover from './Cover';
+import HomePage from './HomePage';
+import StaticPage from './StaticPage';
+
+export default function AdminPreviewStatic({ cover, homepage, opencontent }) {
+  const [ state, setState ] = useState({ isHomepage: true });
+  const clickHandler = () => {
+    setState({
+      ...state,
+      isHomepage: !state.isHomepage
+    });
+  };
+  const StaticHeader = () => (
+    <header className="c-header js-header initialized">
```

Comment:
Since this code has been copied from the rails view, I think the least we can do is to add a comment here and there to remember they must be synced.